src/components/pages/pseudochoice.tsx

```diff
@@ -4,6 +4,46 @@ import React from 'react';
 //   onCorrect: () => void;
 // }
 
+function MoveTurnComponent(input){
```

Comment:
Try to use `props` instead of `input` as it is standard practice. Let me know if you run into any issues doing that.